test/helpers/thesis_helper_test.rb

```diff
@@ -76,4 +76,150 @@ class ThesisHelperTest < ActionView::TestCase
     hybrid_thesis = Thesis.new(degrees: [degrees(:one), degrees(:two)])
     assert_equal true, satisfies_advanced_degree?(hybrid_thesis)
   end
+
+  test 'filter_theses_by_department returns a filtered set of theses' do
+    theses = Thesis.includes(:departments)
+    params[:department] = 'Department of Aeronautics and Astronautics'
+    assert_not_equal theses.count, filter_theses_by_department(theses).count
+  end
+
+  test 'filter_theses_by_department does nothing when no department param is set' do
+    theses = Thesis.includes(:departments)
+    assert_equal theses.count, filter_theses_by_department(theses).count
+  end
+
+  test 'filter_theses_by_department does nothing when department param is "all"' do
+    theses = Thesis.includes(:departments)
+    params[:department] = 'all'
+    assert_equal theses.count, filter_theses_by_department(theses).count
+  end
+
+  test 'filter_theses_by_degree returns a filtered set of theses' do
+    theses = Thesis.includes(:degrees)
+    params[:degree] = 'Master of Fine Arts'
+    assert_not_equal theses.count, filter_theses_by_degree(theses).count
+  end
+
+  test 'filter_theses_by_degree does nothing when no degree param is set' do
+    theses = Thesis.includes(:degrees)
+    assert_equal theses.count, filter_theses_by_degree(theses).count
+  end
+
+  test 'filter_theses_by_degree does nothing when degree param is "all"' do
+    theses = Thesis.includes(:degrees)
+    params[:degree] = 'all'
+    assert_equal theses.count, filter_theses_by_degree(theses).count
+  end
+
+  test 'filter_theses_by_multiple_authors returns a filtered set of theses' do
+    theses = Thesis.all
+    params[:multi_author] = 'true'
+    assert_not_equal theses.count, filter_theses_by_multiple_authors(theses).length
```

Comment:
Similar to another comment, consider whether checking for a smaller set rather than not equal would be more meaningful